TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -456,6 +462,49 @@ class RouteDetailContentViewController: UIViewController {
 
     /// Draw all waypoints initially for all paths in [Path] or [[CLLocationCoordinate2D]], plus fill bounds
     private func drawMapRoute() {
+        var pathCount = 0
+        // Helper function to create bus stop circles
+        func busStopCircles(at coordinate: CLLocationCoordinate2D, on mapView: GMSMapView) -> GMSCircle {
+            let circle = GMSCircle(position: coordinate, radius: 50)
+            circle.fillColor = UIColor.white.withAlphaComponent(1.0)
+            circle.strokeColor = UIColor.black
+            circle.strokeWidth = 2.0
+            circle.map = mapView
+            circle.zIndex = 2
+            return circle
+        }
+        // Helper function to map final location marker
+        func mapLocationMarker() -> UIImage? {
+            let targetSize = CGSize(width: 18, height: 30)
+            guard let originalImage = UIImage(named: "locationMarker") else {
+                return nil
+            }
+            UIGraphicsBeginImageContextWithOptions(targetSize, false, 0.0)
```

Comment:
put a new line after the "}". it breaks it up a little more and is better for readability